runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -402,6 +402,11 @@ public void onChatMessage(ChatMessage event)
 		{
 			createGameTimer(PRAYER_ENHANCE);
 		}
+
+		if (config.showCharge() && event.getMessage().equals("<col=ef1020>You feel charged with magic power.</col>"))
```

Comment:
There's also this when your charge ends, if you want to remove it like other timers do:
![image](https://user-images.githubusercontent.com/35824069/40886351-6fe580f2-6736-11e8-89f0-23a6649f0a09.png)

Reply:
If it doesn't go away before the six minutes are up, there isn't really a need to check for this, as the Timer will be destroyed on its own. I can add it in just in case if you think it would be a good idea.

Reply:
There's some other timers that also remove themselves when a certain message is received so I think it should be consistent with that if possible.